app/components/emoji_picker/index.js

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {connect} from 'react-redux';
+import {createSelector} from 'reselect';
+
+import {getCustomEmojisByName} from 'mattermost-redux/selectors/entities/emojis';
+
+import {getTheme} from 'app/selectors/preferences';
+import {CategoryNames, Emojis, EmojiIndicesByCategory} from 'app/utils/emojis';
+
+import EmojiPicker from './emoji_picker';
+
+function fillEmoji(indice) {
+    return Emojis[indice].aliases[0];
```

Comment:
Maybe we should change this to return the actual emoji objects so that the `EmojiPicker` can see all the aliases